code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -50,18 +50,30 @@
 	var/combo = 0
 	var/combo_time
 	var/combo_wait = 10
+	var/combo_on = TRUE
+
+/obj/item/ego_weapon/despair/attack_self(mob/user)
+	..()
+	if(combo_on)
+		to_chat(user,"<span class='warning'>You swap your grip, and will no longer perform a finisher.</span>")
+		combo_on = FALSE
+		return
+	if(!combo_on)
+		to_chat(user,"<span class='warning'>You swap your grip, and will now perform a finisher.</span>")
+		combo_on =TRUE
+		return
 
 //This is like an anime character attacking like 4 times with the 4th one as a finisher attack.
 /obj/item/ego_weapon/despair/attack(mob/living/M, mob/living/user)
 	if(!CanUseEgo(user))
 		return
-	if(world.time > combo_time)
+	if(world.time > combo_time || !combo_on)	//or you can turn if off I guess
```

Comment:
You basically made it so combo has no time. It does not work as you want it to, at all.

Reply:
I was testing it and it worked just fine in testing